samples/.jshintignore

```diff
@@ -1 +0,0 @@
-*.js
```

Comment:
This file serves no purpose. The `.jshintignore` file at the root of the project is where JSHint determines which files/directories to ignore.